components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -57,6 +57,8 @@
         <jaxrs:properties>
             <!-- This is added to catch interceptor level exceptions in GlobalThrowableMapper. -->
             <entry key="map.cxf.interceptor.fault" value="true" />
+            <!-- This is added to restrict the size of attachments sent through the requests. -->
+            <entry key="attachment-max-size" value="10485760"/>
```

Comment:
Were we able to test with around 11MB? is this config working properly?

Reply:
Yes. It gives a "HTTP/1.1 413 (Payload Too Large)" code.

Reply:
I think we need to have this in all REST APIs